tests/online_ddl/run.sh

```diff
@@ -43,55 +45,62 @@ function real_run() {
-3😊4';" $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl $online_ddl_scheme
+	run_sql_online_ddl "alter table gho_t1 add column c int comment '1  2
+3😊4';" $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl gh-ost
+	run_sql_online_ddl "alter table gho_t2 add column c int comment '1  2
+3😊4';" $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl gh-ost
+	run_sql_online_ddl "alter table gho_t2 add column c int comment '1  2
+3😊4';" $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl gh-ost
+	run_sql_online_ddl "alter table gho_t3 add column c int comment '1  2
+3😊4';" $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl gh-ost
+
+	run_sql_online_ddl "alter table pt_t1 add column c int comment '1  2
+3😊4';" $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl pt
+	run_sql_online_ddl "alter table pt_t2 add column c int comment '1  2
+3😊4';" $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl pt
+	run_sql_online_ddl "alter table pt_t2 add column c int comment '1  2
+3😊4';" $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl pt
+	run_sql_online_ddl "alter table pt_t3 add column c int comment '1  2
+3😊4';" $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl pt
 
 	check_port_offline $WORKER2_PORT 10
 	run_dm_worker $WORK_DIR/worker2 $WORKER2_PORT $cur/conf/dm-worker2.toml
 	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER2_PORT
 
 	# imitate a DM task is started during the processing of online DDL tool
-	if [ "$online_ddl_scheme" == "gh-ost" ]; then
-		run_sql_source1 "rename /* gh-ost */ table online_ddl.ignore to online_ddl._ignore_del, online_ddl._ignore_gho to online_ddl.ignore;"
-	elif [ "$online_ddl_scheme" == "pt" ]; then
-		run_sql_source1 "rename table online_ddl.ignore to online_ddl._ignore_old, online_ddl._ignore_new to online_ddl.ignore;"
-	fi
+	run_sql_source1 "rename /* gh-ost */ table online_ddl.gho_ignore to online_ddl._gho_ignore_del, online_ddl._gho_ignore_gho to online_ddl.gho_ignore;"
+	run_sql_source1 "rename table online_ddl.pt_ignore to online_ddl._pt_ignore_old, online_ddl._pt_ignore_new to online_ddl.pt_ignore;"
 
-	run_sql_file_online_ddl $cur/data/db1.increment.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl $online_ddl_scheme
-	run_sql_file_online_ddl $cur/data/db2.increment.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl $online_ddl_scheme
+	run_sql_file_online_ddl $cur/data/gho.db1.increment.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl gh-ost
+	run_sql_file_online_ddl $cur/data/gho.db2.increment.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl gh-ost
+	run_sql_file_online_ddl $cur/data/pt.db1.increment.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1 online_ddl pt
+	run_sql_file_online_ddl $cur/data/pt.db2.increment.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2 online_ddl pt
 
 	for ((k = 0; k < 10; k++)); do
-		run_sql_tidb "show create table online_ddl.t_target"
+		run_sql_tidb "show create table online_ddl.gho_t_target"
```

Comment:
```suggestion
```

these lines are to make sure DM has synced after `alter table XX_tX add column info_json json GENERATED ALWAYS AS (`info`) VIRTUAL;`

so we can only check the latter one.